apps/api/src/beneficiary/beneficiary.controller.ts

```diff
@@ -54,4 +54,13 @@ export class BeneficiaryController {
   async deleteById(@Param('id') id: string) {
     return await this.beneficiaryService.removeBeneficiary(id)
   }
+
+  @Post('delete-many')
+  @Roles({
+    roles: [RealmViewSupporters.role, ViewSupporters.role],
+    mode: RoleMatchingMode.ANY,
+  })
+  async deleteMany(@Body() data: string[]) {
```

Comment:
An idea for future improvement is to extract the `data: string[]` in own `DeleteBeneficiaryDto` to make sure validation is applied to the body before it hits the serivce.

Reply:
Done.